atomkraft/cli/__init__.py

```diff
@@ -97,4 +97,12 @@ def reactor(
     """
     actions = [act.strip() for act in actions.split(",")]
     variables = [var.strip() for var in variables.split(",")]
+
+    if Path(path.name).is_file():
+        print(
+            f"WARNING: Stub file already exists and it will be overwritten: {path.name}"
+        )
```

Comment:
Can we use [warnings](https://docs.python.org/3/library/warnings.html) module instead?

Reply:
I just removed the warning and now simply display a message to ask the user to continue. I don't think using the `warnings` package was worth it in this case.